e2e/cypress/integration/bot_accounts/promote_demote_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @bot_accounts
+
+import {createBotPatch} from '../../support/api/bots';
+
+describe('Managing bots in Teams and Channels', () => {
+    let team;
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableBotAccountCreation: true,
+            },
+            TeamSettings: {
+                RestrictCreationToDomains: 'sample.mattermost.com',
+            },
+        });
+        cy.apiInitSetup({loginAfter: true}).then((out) => {
+            team = out.team;
+        });
+    });
+    it('MM-T1819 Promote a BOT to team admin', () => {
+        cy.makeClient().then(async (client) => {
+            // # Go to channel
+            const channel = await client.getChannelByName(team.id, 'town-square');
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+
+            // # Add bot to team
+            const bot = await client.createBot(createBotPatch());
+            await client.addToTeam(team.id, bot.user_id);
+
+            // # Open member invite screen
+            cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+            cy.get('#manageMembers > button').should('contain.text', 'Manage Members').click();
+
+            // # Find bot
+            cy.get('#searchUsersInput').type(`${bot.username}`);
```

Comment:
Sometimes the filter input displays before the modal list and typing gets cut-off. Just needed to make sure modal list is loaded before search is performed.

```suggestion
            cy.get('.more-modal__list').find('.more-modal__row').its('length').should('be.gt', 0);
            cy.get('#searchUsersInput').type(`${bot.username}`);
```